app/models/degree.rb

```diff
@@ -0,0 +1,3 @@
+class Degree < ApplicationRecord
+  validates :type_of_degree, presence: true
```

Comment:
I don't love method names in the pattern `thing.thing_type`, but maybe this was to avoid the reserved word `type`?

How about `degree.category` or `degree.classification`?

Reply:
Yes, `type` is a reserved word.
Generally, I agree with you, but this time I think `type_of_degree` is the exact copy present in the page, and we do not need to think what category/classification correspond to. 

<img width="806" alt="Screenshot 2019-07-29 at 10 08 24" src="https://user-images.githubusercontent.com/58971/62036136-1e93a680-b1e9-11e9-9c19-4e38b992463f.png">